node/node.go

```diff
@@ -256,6 +257,21 @@ func (node *Node) tryBroadcast(tx *types.Transaction) {
 	}
 }
 
+func (node *Node) tryBroadcastStaking(stakingTx *staking.StakingTransaction) {
+	msg := proto_node.ConstructStakingTransactionListMessageAccount(staking.StakingTransactions{stakingTx})
+
+	shardGroupID := nodeconfig.NewGroupIDByShardID(nodeconfig.ShardID(0)) // broadcast to beacon chain
+	utils.Logger().Info().Str("shardGroupID", string(shardGroupID)).Msg("tryBroadcastStaking")
+
+	for attempt := 0; attempt < NumTryBroadCast; attempt++ {
```

Comment:
will it causing problem to resend the same staking tx multiple times in retry?  will it create multiple validators on the same account?

Reply:
No, the p2p network will screen that out if it's the same message. Also transactions can't be repeated because of nonce.